src/forder.c

```diff
@@ -418,9 +418,12 @@ SEXP forder(SEXP DT, SEXP by, SEXP retGrpArg, SEXP sortGroupsArg, SEXP ascArg, S
   memset(stat,   0, 257*sizeof(uint64_t));
   TBEG()
 #endif
+  
+  if ((!isLogical(verbose) && !isInteger(verbose)) || LENGTH(verbose)!=1 || INTEGER(verbose)[0]==NA_INTEGER)
+    error(_("verbose option must be length 1 non-NA logical or integer"));
 
   int n_protect = 0;
-  const bool verbose = GetVerbose();
+  const int Verbose = INTEGER(verbose)[0];
```

Comment:
Not sure if it's good practice to just capitalize the first letter of a variable name for a separate type of variable (bools vs int in this case)

Reply:
the thing is that i actually didn't wanted to change the variable name, but needed to name the arguement with same name `verbose` hence thought of changing the variable name instead and wanted something that doesn't confuse people about what this variable's role is ? so thought `Variable ` would be fitting, what else could be better here ?

Reply:
Our common practice would be to rename the function argument to `verboseArg` and then use `verbose`